contracts/uninsured-vesting-v1/UninsuredVestingV1.sol

```diff
@@ -67,19 +68,16 @@ contract UninsuredVestingV1 is Ownable {
         if (totalAllocated == 0) revert NoAllocationsAdded();
 
         startTime = block.timestamp;
-
         uint256 delta = totalAllocated - Math.min(xctd.balanceOf(address(this)), totalAllocated);
-
         xctd.safeTransferFrom(msg.sender, address(this), delta);
 
         emit StartTimeSet(startTime);
     }
 
     function claim(address target) public {
+        if (!(msg.sender == owner() || msg.sender == target)) revert OnlyOwnerOrSender();
```

Comment:
What happened to the modifyer here?

Reply:
I didn't use it since in the uninsured contract there's only one place that requires this (we only have the modifier in the insured contract)

Reply:
Ah, my bad. Thought this was insured contract!